theano/scan_module/tests/test_scan_checkpoint.py

```diff
@@ -6,25 +6,25 @@
 import theano
 import theano.tensor as T
 
+from pygpu.gpuarray import GpuArrayException
```

Comment:
I think moving the import in the test (with a comment to explain) would fix the problem in travis or doing a try except ImportError: pass around that import.